### DISCUSSION TITLE
Preview Track will not be recyle(not end) after its finished playing

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#19577] [Q&A] [answered]
I'm not sure it is a performace problem but it seens like lazer never try to recycle them.
But when the preview of the same song is played again, it will be reused.
Maybe to prevent downloading again?
Should a time limit be set for recycling?
in the video, after playing the preview music, `Track` stat didn't decrease

https://user-images.githubusercontent.com/34775378/182855621-87baebeb-0228-4028-aefa-0d2128bd28fc.mp4

**Comment by peppy** [comment]
Hvae you tried forcing a "clear cache" operation in settings?

**Reply by cdwcgt** [reply]
"clear cache" doesn't work

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/19581
